apis/types/task_create_request.go

```diff
@@ -16,7 +16,13 @@ import (
 // TaskCreateRequest task create request
 // swagger:model TaskCreateRequest
 type TaskCreateRequest struct {
+	TaskCreateRequestBase
```

Comment:
Could you please guarantee the compatibility between the `dfget 0.3.x` and the `latest supernode`.